app/controllers/supervisor/user_exams_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Supervisor::UserExamsController < SupervisorController
+  before_action :load_trainee, only: :index
+
+  def index
+    @user_exams = @trainee.user_exams
+                          .sort_by_created_at_desc
+                          .includes({exam: :exam_questions}, :subject)
+                          .paginate(page: params[:page])
+                          .per_page(Settings.page)
+  end
+
+  private
+
+  def load_trainee
+    @trainee = User.find_by id: params[:trainee_id]
```

Comment:
Bạn có thể viết như thế này để trông clean hơn
```suggestion
    @trainee = User.find_by_id params[:trainee_id]
```

Reply:
Theo quy ước của clean code trên CI của Framgia-CI thì `@trainee = User.find_by id: params[:trainee_id]` clean hơn ạ